### DISCUSSION TITLE
Where to render and load the data from an API.

### DISCUSSION THREAD

**Discussion by maaz-bin-asad** [react-page/react-page#1048] [Q&A]
Suppose I have posted the data to database. Now I wish to load from an API and render it when user comes to the page next time. Where do we call the API and render the data?

**Comment by macrozone** [comment]
@maaz-bin-asad that depends on the rest of your project, the used framework, etc. I think this question is very broad and ist not about react-page per se. There is nothing special about react-page. Consider it like a form-field:

`<input type="text" value={valueFromDB} onChange={saveValue} />` 

You can use react-page identically
